src/Console/Command/BuildCommand.php

```diff
@@ -47,6 +47,7 @@ protected function configure()
                 new InputOption('repository-url', null, InputOption::VALUE_OPTIONAL, 'Only update the repository at given url', null),
                 new InputOption('no-html-output', null, InputOption::VALUE_NONE, 'Turn off HTML view'),
                 new InputOption('skip-errors', null, InputOption::VALUE_NONE, 'Skip Download or Archive errors'),
+                new InputOption('best-candidate-strategy', null, InputOption::VALUE_NONE, 'Download only the best candidate version'),
```

Comment:
it is better than adding more CLI flags that you must pass each time you build to get the same output (btw, this flag would also impact the prune command, which is also why having it in the config file is better)